tests/unittests/main.cpp

```diff
@@ -37,6 +37,9 @@ using namespace librepcb;
  ******************************************************************************/
 
 int main(int argc, char* argv[]) {
+  // Initialize a common locale for all tests
```

Comment:
Ah, minor nitpick: The rest of the comments start with a lowercase letter, maybe that could be kept for consistency.